Makefile

```diff
@@ -69,6 +69,9 @@ endif
 
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
+# shell command used for `make shell`
+SH := $(if $(SH),$(SH),bash)
```

Comment:
Yes, but for consistency with other parameters (`DOCKER_GRAPHDRIVER`. `PKGCACHE_DIR`,`DOCKER_INCREMENTAL_BINARY`..), I think keeping the current style is ok, WDYT?